### ISSUE TITLE
Allow for initial context to be read from scenario directly

### ISSUE THREAD

**Issue by robert-stuttaford** [mhjort/clj-gatling#32]
So that request functions can be written in a reusable fashion.

E.g.

``` clojure
:scenarios [{:name "..." :steps [...] :context {:product-slug "my-product-slug"}}]

(defn product-request [{:keys [product-slug] :as context}]
   ... use product-slug to produce a url for request ...)
```

**Comment by mhjort** [comment]
You can already specify ```:context``` at global level using options. But it would be good to have support for specifying that at scenario level too as you suggested.

**Comment by mhjort** [comment]
Implemented in ```0.8.5``` Thanks for @greywolve !
